charger/sungrow.go

```diff
@@ -142,12 +142,13 @@ func (wb *Sungrow) Status() (api.ChargeStatus, error) {
 
 // Enabled implements the api.Charger interface
 func (wb *Sungrow) Enabled() (bool, error) {
-	b, err := wb.conn.ReadInputRegisters(sgRegStartMode, 1)
+	b, err := wb.conn.ReadInputRegisters(sgRegState, 1)
 	if err != nil {
 		return false, err
 	}
 
-	return binary.BigEndian.Uint16(b) != 0, nil
+	state := binary.BigEndian.Uint16(b)
+	return state > 0 && state < 6, nil
```

Comment:
```suggestion
	return binary.BigEndian.Uint16(b) < 6, nil
```

Reply:
Sorry, actually did an error here, fixed in new commit, should be > 1 && < 6